internal/executor/logger/zerolog.go

```diff
@@ -0,0 +1,64 @@
+package logger
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"octavius/internal/pkg/constant"
+	"os"
+	"time"
+)
+
+//Logger holds the pointer to zerolog.Logger object
+type Logger struct {
+	logger *zerolog.Logger
+}
+
+var log Logger
+
+//Setup intializes the logger object
+func Setup(configLogLevel string) {
+	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+	f, _ := os.OpenFile("auditLogs.txt", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	multi := zerolog.MultiLevelWriter(f, consoleWriter)
+	logLevel, err := zerolog.ParseLevel(configLogLevel)
+
+	if err != nil {
+		fmt.Println("log level parsing problem")
```

Comment:
if there's an error in log level parsing, manually setting the log level to info is a better idea than printing the log level parsing problem, otherwise we may encounter issue in loginit initialisation in next line (because logLevel might not be found in the case there's an error).